worker/src/RTC/Codecs/VP8.cpp

```diff
@@ -200,6 +200,12 @@ namespace RTC
 		{
 			MS_TRACE();
 
+			if (data == nullptr)
+			{
+				MS_ERROR("data is nullptr");
+				return;
+			}
```

Comment:
```suggestion
			if (data == nullptr)
			{
				MS_ERROR("data is nullptr");

				return;
			}
```

But honestly we never use `MS_ERROR` for these things. This should be `MS_ABORT` instead.